src/cascade/dismod/serialize.py

```diff
@@ -506,28 +527,52 @@ def smooth_id_func(smooth):
     return smooth_table, grid_table, smooth_id_func
 
 
-def make_rate_table(context, smooth_id_func):
+def make_rate_and_nslist_tables(context, smooth_id_func, location_to_node_func):
```

Comment:
Ok, I'll comment. The short answer is because if I do them together I can do it in one fairly simple pass but if I separate them I need more passes and more complexity (I think).